lib/yt/audit.rb

```diff
@@ -36,7 +37,8 @@ def run
         Yt::VideoAudit::SubscribeEndScreen.new(videos: @videos),
         Yt::VideoAudit::VideoEndScreen.new(videos: @videos),
         Yt::VideoAudit::PlaylistEndScreen.new(videos: @videos),
-        Yt::VideoAudit::WebsiteEndScreen.new(videos: @videos)
+        Yt::VideoAudit::WebsiteEndScreen.new(videos: @videos),
+        Yt::VideoAudit::InvideoProgramming.new(videos: @videos)
```

Comment:
@kangkyu If you add a comma at the end then next time we add a line, our `git diff` will only include one line.